pgjdbc/src/main/java/org/postgresql/jdbc/PgSQLInput.java

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright (c) 2004, PostgreSQL Global Development Group
```

Comment:
2024

Reply:
I just copied the header from PgResultSet.java.  :)

Reply:
So the protocol here is to copyright it the year the file was created I guess?

Reply:
Yes, year it was created